<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="scene_config"> 
  <!-- Distance between the two robot bases -->
  <!-- Left arm base is at the world origin -->
  <xacro:property name="right_arm_x" value="2.0"/>

  <!-- Pose and size of additional objects -->
  <xacro:property name="lift_box_x" value="1.0"/>
  <xacro:property name="lift_box_y" value="0.0"/>
  <xacro:property name="lift_box_z" value="0.0"/>
  <xacro:property name="lift_box_roll" value="0.0"/>
  <xacro:property name="lift_box_pitch" value="0.0"/>
  <xacro:property name="lift_box_yaw" value="0.0"/>

  <xacro:property name="lift_box_size_x" value="0.3"/>
  <xacro:property name="lift_box_size_y" value="0.2"/>
  <xacro:property name="lift_box_size_z" value="0.2"/>

</robot>

